packages/canon/src/components/Button/styles.css

```diff
@@ -14,44 +14,60 @@
  * limitations under the License.
  */
 
-:root {
-  --button-primary-background-color: #1ed760;
-  --button-primary-background-color-hover: #3be477;
-  --button-primary-background-color-active: #1abc54;
-  --button-primary-border-color: #1ed760;
-  --button-primary-border-color-hover: #3be477;
-  --button-primary-border-color-active: #1abc54;
-  --button-primary-text-color: #000;
-  --button-primary-border-radius: 9999px;
-  --button-secondary-background-color: transparent;
-  --button-secondary-background-color-hover: transparent;
-  --button-secondary-background-color-active: rgba(0, 0, 0, 0.03);
-  --button-secondary-border-color: #000;
-  --button-secondary-border-color-hover: #000;
-  --button-secondary-border-color-active: #000;
-  --button-secondary-text-color: #000;
-  --button-secondary-border-radius: 9999px;
-  --button-size-small: 32px;
-  --button-size-medium: 48px;
-}
-
-[data-theme='dark'] {
-  --button-primary-background-color: #1ed760;
-  --button-primary-background-color-hover: #3be477;
-  --button-primary-background-color-active: #1abc54;
-  --button-primary-border-color: #1ed760;
-  --button-primary-border-color-hover: #3be477;
-  --button-primary-border-color-active: #1abc54;
-  --button-primary-text-color: #000;
-  --button-primary-border-radius: 9999px;
-  --button-secondary-background-color: transparent;
-  --button-secondary-background-color-hover: transparent;
-  --button-secondary-background-color-active: rgba(255, 255, 255, 0.1);
-  --button-secondary-border-color: #fff;
-  --button-secondary-border-color-hover: #fff;
-  --button-secondary-border-color-active: #fff;
-  --button-secondary-text-color: #fff;
-  --button-secondary-border-radius: 9999px;
-  --button-size-small: 32px;
-  --button-size-medium: 48px;
+.button {
```

Comment:
Managing API stability and evolution of CSS styles is a big unknown for us. Don't wanna block the individual component PRs but there's definitely a need for exploration in this area

Reply:
Yes this is really an alpha for now. A lot will change until we solve all these issues but that's easier to do across multiple components and see what work holistically.